freud/_box.pxd

```diff
@@ -48,6 +48,11 @@ cdef extern from "Box.h" namespace "freud::box":
                     unsigned int) const
         vec3[float] centerOfMass(vec3[float]*, size_t, float*) const
         void center(vec3[float]*, size_t, float*) const
+        void computeDistances(vec3[float]* points, vec3[float]* query_points,
```

Comment:
We haven't named the arguments elsewhere in this definition file, so I think you can remove the names `points`, etc. for consistency.